pages/dkp/kommander/2.1/install/configuration/index.md

```diff
@@ -75,6 +75,10 @@ apps:
 ...
 ```
 
+### Minimal Kommander installation
```

Comment:
Since we apparently don't list 3rd-level headings in the right-hand navigation, I would recommend making it a 2nd-level heading for users to find it quicker.